pkg/util/pravegacluster.go

```diff
@@ -81,6 +82,18 @@ func ConfigMapNameForSegmentstore(clusterName string) string {
 	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
 }
 
+func SetConfigMapNameForSSEnvVars(name string, flag bool) {
```

Comment:
These setter/getter methods are not needed.

Reply:
Besides its incorrect to use "global" variables like ssEnvVar in operator code as it would interfere with managing multiple Pravega Clusters.